src/promptflow/promptflow/_cli/_pf/_flow.py

```diff
@@ -287,7 +288,8 @@ def _init_existing_flow(flow_name, entry=None, function=None, prompt_params: dic
     python_tool_inputs = [arg.name for arg in python_tool.tool_arg_list]
     for tool_input in tools.prompt_params.keys():
         if tool_input not in python_tool_inputs:
-            raise ValueError(f"Template parameter {tool_input} doesn't find in python function arguments.")
+            error = ValueError(f"Template parameter {tool_input} doesn't find in python function arguments.")
+            raise UserErrorException(target=ErrorTarget.CONTROL_PLANE_SDK, message=str(error), error=error)
```

Comment:
I think we have SDKERROR class shall we use that?

Reply:
If `SDKError `is used, the `error type` will become `SDKError `instead of `ValueError`
But I think it's better to keep the `error type` as `ValueError`?
![image](https://github.com/microsoft/promptflow/assets/23182548/e8f7b44f-13b1-4b32-93cb-b4d4d48c61a3)